.github/workflows/python-package.yml

```diff
@@ -0,0 +1,87 @@
+name: Python package
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  schedule:
+  - cron: 1 0 * * *  # Run daily at 0:01 UTC
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
FTR if you want, it's possible to add more OSs by parametrizing the matrix as follows: https://github.com/pypa/setuptools/blob/df9157c/.github/workflows/python-tests.yml#L27-L32

Reply:
Nice, I'm good with just ubuntu for now.